src/services/expenses.service.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+let expenses = [];
+let countIdExpenses = 1;
+
+const getAllExpenses = ({ userId, categories, from, to }) => {
+  let filteredExpenses = [...expenses];
+
+  if (userId) {
+    filteredExpenses = filteredExpenses
+      .filter(expense => expense.userId === +userId);
+  }
+
+  if (categories) {
+    filteredExpenses = filteredExpenses
+      .filter(expense => categories.includes(expense.category));
+  }
+
+  if (from && to) {
+    filteredExpenses = filteredExpenses
+      .filter(expense => expense.spentAt >= from && expense.spentAt <= to);
+  }
+
+  return filteredExpenses;
+};
+
+const getExpenseById = (expenseId) => {
+  const foundExpense = expenses.find(expense => expense.id === +expenseId);
+
+  return foundExpense || null;
```

Comment:
<img width="781" alt="image" src="https://github.com/mate-academy/node_accounting-app-v2/assets/54860641/5573b8a8-06ca-487f-bb41-b1d60c2e9e99">

```suggestion
  return expenses.find(expense => expense.id === +expenseId);
```